### ISSUE TITLE
Any update for Python 3.9?

### ISSUE THREAD

**Issue by Nirvanatin** [intxcc/pyaudio_portaudio#40]
Is there any update or alternative for Python 3.9?

**Comment by hubert-gabryel** [comment]
Hi All, 
We are waiting for feedback

**Comment by s0d3s** [comment]
Hello, maybe [PyAudioWPatch](https://github.com/s0d3s/PyAudioWPatch) is for you. Available for Windows **x64/x32** Python 3.{7-11}

Everything is based on the latest releases of **PyAudio** and **PortAudio**. Loopback support is now more canonical.

**Comment by pradsy90** [comment]
Yep, PyAudioWatch works like a charm. THANK YOU! I did a sample using PyAudioWatch on Python 3.10.10 and worked good!

**Comment by Nirvanatin** [comment]
I get an empty 1KB recording file after running the pawp_record_wasapi_loopback.py
Am I missing something?
